.github/workflows/ci.yml

```diff
@@ -2,6 +2,7 @@ name: CI
 on:
   push:
   workflow_dispatch:
+  workflow_call:
```

Comment:
Unrelated template update

Reply:
This is a (not very important) issue with the cookiecutter: `make template` both applies any changes that you've made in the `.cookiecutter/` directory (such as the `cookiecutter.json` change in this PR) and also upgrades you to the latest version of the cookiecutter. I think it'd be nice if we recorded in `cookiecutter.json` the cookiecutter commit that was last used and then `make template` could use `bin/make_template`'s `checkout` option to replay the same version of the cookiecutter, or `make template upgrade=yes` could upgrade the commit in `cookiecutter.json` and then replay the template